virtualizarr/manifests/array.py

```diff
@@ -131,7 +131,7 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs) -> Any:
             return _isnan(self.shape)
         return NotImplemented
 
-    def __array__(self) -> np.ndarray:
+    def __array__(self, dtype: np.typing.DTypeLike = None) -> np.ndarray:
```

Comment:
Have to match the base class's signature (but we raise NotImplementedError anyway, so it doesn't matter)